esphome/components/remote_base/__init__.py

```diff
@@ -265,6 +266,53 @@ async def build_dumpers(config):
     return dumpers
 
 
+# Beo4
+Beo4Data, Beo4BinarySensor, Beo4Trigger, Beo4Action, Beo4Dumper = declare_protocol(
+    "Beo4"
+)
+BEO4_SCHEMA = cv.Schema(
+    {
+        cv.Required(CONF_SOURCE): cv.hex_uint8_t,
+        cv.Required(CONF_COMMAND): cv.hex_uint8_t,
+        cv.Optional(CONF_COMMAND_REPEATS, default=1): cv.hex_uint8_t,
```

Comment:
```suggestion
        cv.Optional(CONF_COMMAND_REPEATS, default=1): cv.uint8_t,
```
Wouldn't this be better?

Reply:
Yes, that sounds reasonable. I will fix that.